tests/opensearch/basic.bats

```diff
@@ -12,17 +12,11 @@ load "$BATS_ASSERT_ROOT/load.bash"
 load "$BATS_FILE_ROOT/load.bash"
 
 function setup_file() {
+    export TEST_NAMESPACE=${TEST_NAMESPACE:-opensearch-basic}
```

Comment:
Will this impact future tests or is it a sub-shell?

Reply:
sub-shell: 

`bats_run_tests_in_parallel` from [here](https://github.com/bats-core/bats-core/blob/de96df03197ecc51635463fd9e35e26638191a90/libexec/bats-core/bats-exec-file#L246) calls `bats_semaphore_run` which creates the sub-shell [here](https://github.com/bats-core/bats-core/blob/de96df03197ecc51635463fd9e35e26638191a90/lib/bats-core/semaphore.bash#L42-L48)

Reply:
Ah good, so `export` does not compromise future runs then.
Can you add a comment so it is clear and, if it changes in the future, we know where to look?

Reply:
This is defined as part of the bats documentation [here](https://bats-core.readthedocs.io/en/stable/writing-tests.html#setup-and-teardown-pre-and-post-test-hooks) 

> Variables that are exported in setup_file will be visible to all following functions (setup, the test itself, teardown, teardown_file).

Do you want a comment with the link added near every setup_file or do you just want this linked in this repos readme/contributing docs?